Mathlib/Analysis/Complex/Polynomial.lean

```diff
@@ -77,8 +77,8 @@ theorem card_complex_roots_eq_card_real_add_card_not_gal_inv (p : ℚ[X]) :
   rw [← Finset.card_image_of_injective _ Subtype.coe_injective, ←
     Finset.card_image_of_injective _ inj]
   let a : Finset ℂ := ?_
-  let b : Finset ℂ := ?_
-  let c : Finset ℂ := ?_
+  on_goal 1 => let b : Finset ℂ := ?_
+  on_goal 1 => let c : Finset ℂ := ?_
```

Comment:
Perhaps the linter shouldn't add `on_goal` when the additional goals are terms of `Type` and not `Prop`.

Reply:
This is probably a good heuristic, though I would limit the use of this heuristic to theorems.  Again, I'll leave this up for further comments.